SessionUIKit/Style Guide/Themes/Theme+Colors.swift

```diff
@@ -115,10 +115,10 @@ internal extension UIColor {
     static let oceanLight7: UIColor = #colorLiteral(red: 0.9882352941, green: 1, blue: 1, alpha: 1)        // #FCFFFF
 }
 
-public extension UIColor {
-    /// This value shouldn't be used dirrectly, it should be resolved via the `ThemeManager.color(for:in:with:)` function, due
-    /// to this the _actual_ colour value is set to an obvious colour to help indicate incorrect usage
-    static let primary: UIColor = UIColor(red: 1, green: 0, blue: 1, alpha: 1)
+internal extension UIColor {
+    static let primary: UIColor = UIColor(dynamicProvider: { _ in
+        return ThemeManager.primaryColor.color
+    })
```

Comment:
So this change was actually intentional as the theme system doesn't work properly in some cases unless `ThemeManager.color(for:in:with:)` is used to resolve the primary colour

It's using `UIColor(red: 1, green: 0, blue: 1, alpha: 1)` in order to make it very obvious when something isn't using the system correctly - was there a specific section which broke due to this change? (as it likely means there is a bug in either `UIKit+Theme` or  `SwiftUI+Theme`)